### ISSUE TITLE
No obvious link to GitHub from the documentation

### ISSUE THREAD

**Issue by stefanor** [mdtraj/mdtraj#1927]
I found the GitHub URL in the installation instructions. A prominent link on the front page would be helpful.

**Comment by mattwthompson** [comment]
Does the "Edit on GitHub" button on the [landing page](https://mdtraj.readthedocs.io/en/latest/) bring you to the repo?

**Comment by stefanor** [comment]
Yes, but the equivalent link on mdtraj.org doesn't link to GitHub.

**Comment by mattwthompson** [comment]
That website doesn't have the most recent documentation, though we're trying to redirect from it to the official website at [mdtraj.readthedocs.io/en/latest/](https://mdtraj.readthedocs.io/en/latest/) https://github.com/mdtraj/mdtraj/issues/1810
